packages/react-native-sdk/docusaurus/docs/reactnative/02-tutorials/01-video-calling.mdx

```diff
@@ -3,833 +3,277 @@ title: Video Call Tutorial
-
-[//]: # (    <StreamVideo client={client}>)
-
-[//]: # (      <StreamCall call={call}>)
-
-[//]: # (        <StreamTheme className="video__call">)
-
-[//]: # (          <UI />)
-
-[//]: # (        </StreamTheme>)
-
-[//]: # (      </StreamCall>)
-
-[//]: # (    </StreamVideo>)
-
-[//]: # (  &#41;;)
-
-[//]: # (})
-
-[//]: # ()
-[//]: # (export const UI = &#40;&#41; => {)
-
-[//]: # (  return &#40;)
-
-[//]: # (    <>)
-
-[//]: # (      <PaginatedGridLayout />)
-
-[//]: # (      <div style={{ display: 'flex' }}>)
-
-[//]: # (        <ToggleAudioPublishingButton />)
-
-[//]: # (        <ToggleVideoPublishingButton />)
-
-[//]: # (      </div>)
-
-[//]: # (    </>)
-
-[//]: # (  &#41;;)
-
-[//]: # (};)
-
-[//]: # (```)
-
-[//]: # ()
-[//]: # ([Theming]&#40;../../ui-components/video-theme&#41; gives you control over the colors, border-radius settings, and icons used by the built-in UI components:)
-
-[//]: # ()
-[//]: # (```css)
-
-[//]: # (.str-video {)
-
-[//]: # (  --str-video__primary-color: #036c5f;)
-
-[//]: # (  --str-video__secondary-color: #4fb9af;)
-
-[//]: # (  --str-video__border-radius-xs: 5px;)
-
-[//]: # (  --str-video__icon--camera: url&#40;'base64 encoded SVG'&#41;;)
-
-[//]: # (})
-
-[//]: # (```)
-
-[//]: # ()
-[//]: # (### Recap)
-
-[//]: # ()
-[//]: # (Please do let us know if you ran into any issues while building an video calling app with React. Our team is also happy to review your UI designs and offer recommendations on how to achieve it with Stream.)
-
-[//]: # ()
-[//]: # (To recap what we've learned about Stream video calling:)
-
-[//]: # ()
-[//]: # (- You set up a call: &#40;`const call = client.call&#40;"default", "123"&#41;`&#41;)
-
-[//]: # (- The call type &#40;"default" in the above case&#41; controls which features are enabled and how permissions are setup)
-
-[//]: # (- When you join a call, real-time communication is set up for audio & video calling: &#40;`call.join&#40;&#41;`&#41;)
-
-[//]: # (- State-related hooks such as `useLocalParticipant` make it easy to build your own UI)
-
-[//]: # (- `ParticipantView` is the low-level component that renders audio and video)
-
-[//]: # ()
-[//]: # (We've used [Stream's Video Calling API]&#40;https://getstream.io/video/&#41;, which means calls run on a global edge network of video servers. By being closer to your users the latency and reliability of calls are better. The React SDK enables you to build in-app video calling, audio rooms and livestreaming in days.)
-
-[//]: # ()
-[//]: # (We hope you've enjoyed this tutorial and please do feel free to reach out if you have any suggestions or questions.)
+Stream ships with several UI components to make this easy. You can customize the components with theming, props and swapping parts of them. This is convenient if you want to quickly build a production-ready calling experience for your app. (and if you need more flexibility, many customers use the above low-level approach to build a UI from scratch).
```

Comment:
We should link to some of the theme-ing pages so that readers know where to look for when they want to do that.